polaris/evaluate/metrics/docking_metrics.py

```diff
@@ -0,0 +1,134 @@
+# This script includes docking related evaluation metrics.
+
+from typing import Union, List
+from copy import deepcopy
+
+import numpy as np
+from rdkit.Chem.rdMolAlign import CalcRMS, GetBestRMS
+
+import datamol as dm
+
+
+## The following rmsd implementation are modified based on ppsebusters https://github.com/maabuu/posebusters/blob/main/posebusters/modules/rmsd.py
+
+
+def _call_rdkit_rmsd(mol_probe: dm.Mol, mol_ref: dm.Mol, conf_id_probe: int, conf_id_ref: int, **params):
+    try:
+        return _rmsd(mol_probe, mol_ref, conf_id_probe, conf_id_ref, **params)
+    except RuntimeError:
+        pass
+    except ValueError:
+        pass
+
+    return np.nan
+
+
+def _rmsd(
+    mol_probe: dm.Mol, mol_ref: dm.Mol, conf_id_probe: int, conf_id_ref: int, kabsch: bool = False, **params
+):
+    if kabsch is True:
+        return GetBestRMS(prbMol=mol_probe, refMol=mol_ref, prbId=conf_id_probe, refId=conf_id_ref, **params)
+    return CalcRMS(prbMol=mol_probe, refMol=mol_ref, prbId=conf_id_probe, refId=conf_id_ref, **params)
+
+
+def robust_rmsd(
+    mol_probe: dm.Mol,
+    mol_ref: dm.Mol,
+    conf_id_probe: int = -1,
+    conf_id_ref: int = -1,
+    drop_stereo: bool = False,
+    heavy_only: bool = True,
+    kabsch: bool = False,
+    symmetrizeConjugatedTerminalGroups=True,
+    **params,
+) -> float:
+    """RMSD calculation for isomers."""
+    mol_probe = deepcopy(mol_probe)
+    mol_ref = deepcopy(mol_ref)  # copy mols because rdkit RMSD calculation aligns mols
```

Comment:
**Suggestion:** Use `dm.copy_mol`. It does exactly the same, but that way if we ever improve the method somehow on the datamol side, we benefit from those changes here. 

```suggestion
    mol_probe = dm.copy_mol(mol_probe)
    mol_ref = dm.copy_mol(mol_ref)  # copy mols because rdkit RMSD calculation aligns mols
```